pages/osx/security.md

```diff
@@ -13,4 +13,12 @@
 
 - Create a keychain:
 
-`security create-keychain -p {{password}} {{name.keychain}}`
+`security create-keychain -p {{password}} {{keychain.name}}`
+
+- Set a preferred certificate to use with a website or [s]ervice by its [c]ommon name (note that this will fail if several certificates with the same common name exist):
+
+`security set-identity-preference -s {{URL, hostname, or service}} -c {{friendly certificate name in keychain}} {{path such as /Library/Keychains/System.keychain}}`
```

Comment:
```suggestion
- Set a certificate to use with a website or [s]ervice by its [c]ommon name (this will fail if several certificates with the same common name exist):

`security set-identity-preference -s {{URL | hostname | service}} -c {{friendly_certificate_name_in_keychain}} {{path/to/file.keychain}}`
```